example-project-with-subdir/tests/capi.rs

```diff
@@ -0,0 +1,19 @@
+#[cfg(feature = "capi")]
+mod capi {
+
+    use inline_c::assert_c;
+
+    #[test]
+    fn test_capi() {
+        (assert_c! {
+        #include <example_project_with_subdir.h>
```

Comment:
You should use `"` not `<` `>`. Same everywhere else.

Reply:
But why?

AFAIU, using `"` allows including files relative to the source file, but this is not what I want to test.

I want to test the real usage, where the included file is installed in some place, not next to the test file.

See https://en.cppreference.com/w/cpp/preprocessor/include.